src/plays/drawing-board/index.css

```diff
@@ -0,0 +1,52 @@
+@import "../../../node_modules/xp.css/dist/XP.css";
```

Comment:
Can't we use the absolute path?

Reply:
I tried using `@import"~xp.css/dist/XP.css";` but was getting error hence had to use absolute path

Reply:
it actually worked! changing and commiting